src/v0/destinations/google_adwords_offline_conversions/utils.js

```diff
@@ -346,6 +346,26 @@ const populateUserIdentifier = ({ email, phone, properties, payload, UserIdentif
   }
   return copiedPayload;
 };
+
+/**
+ * remove redundant ids
+ * @param {*} conversionCopy
+ */
+const updateConversion = (conversion) => {
```

Comment:
> const updateConversion = (conversion) => {
  const copy = cloneDeep(conversion);
  if (copy.gclid) {
    delete copy.wbraid;
    delete copy.gbraid;
  } else if (copy.wbraid && copy.gbraid) {
    throw new InstrumentationError("You can't use both wbraid and gbraid.");
  } else if (copy.wbraid || copy.gbraid) {
    delete copy.userIdentifiers;
  }
  return copy;
};


can we use this as all are mutually exclusive conditions